README.md

```diff
@@ -81,4 +86,30 @@ Clink uses [ClangFormat](https://clang.llvm.org/docs/ClangFormat.html) to format
 find . \( -name "*.cc" -or -name "*.h" \) -not -path "./tfrt/*" -exec clang-format -i -style=llvm {} \;
 ```
 
+### Format java code using diffplug-spotless
+
+Clink uses [diffplug/spotless](https://github.com/diffplug/spotless) to format java code.
+
+```
+cd java-lib
+mvn spotless:apply
+```
+
+### Package java code into jar
+
+```
+bazel build //:clink_kernels_java
+```
+
+### View Java Code with IDE
+
+Clink provides maven configuration that allows uses to view or edit java code with IDEs like IntelliJ IDEA. Before IDEs can correctly compile java project, users need to run the following commands after setting up Clink repo and build Clink.
+
+```
+bazel build //:OneHotEncoderJava
+cp bazel-bin/libOneHotEncoder-speed.jar java-lib/lib/
+```
+
+Then users can open IDEs on `java-lib` directory.
 
+Note: current maven configuration only allows viewing or editing Java code in IDEs. It is not guaranteed that the project can run correctly without bazel.
```

Comment:
Since the title says `View Java Code with IDE`, it does not necessarily mean we can execute Java code in IDE directly. Could we simplify the README by removing this statement?

Reply:
OK. I'll remove this.